addons/product/models/res_config_settings.py

```diff
@@ -57,3 +64,5 @@ def set_values(self):
         if not self.group_discount_per_so_line:
             pl = self.env['product.pricelist'].search([('discount_policy', '=', 'without_discount')])
             pl.write({'discount_policy': 'with_discount'})
+
+        self.env['res.company']._manage_companies_pricelists()
```

Comment:
This should be added in `_activate_group_multi_currency` as well to create pricelists when the pricelist group is enabled ;) (which doesn't only happen through the settings :eyes:)